examples/SPDXJSONExample-v2.0.json

```diff
@@ -2,380 +2,267 @@
+      "extractedText" : "/*\n * (c) Copyright 2000, 2001, 2002, 2003, 2004, 2005, 2006, 2007, 2008, 2009 Hewlett-Packard Development Company, LP\n * All rights reserved.\n *\n * Redistribution and use in source and binary forms, with or without\n * modification, are permitted provided that the following conditions\n * are met:\n * 1. Redistributions of source code must retain the above copyright\n *    notice, this list of conditions and the following disclaimer.\n * 2. Redistributions in binary form must reproduce the above copyright\n *    notice, this list of conditions and the following disclaimer in the\n *    documentation and/or other materials provided with the distribution.\n * 3. The name of the author may not be used to endorse or promote products\n *    derived from this software without specific prior written permission.\n *\n * THIS SOFTWARE IS PROVIDED BY THE AUTHOR ``AS IS'' AND ANY EXPRESS OR\n * IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES\n * OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED.\n * IN NO EVENT SHALL THE AUTHOR BE LIABLE FOR ANY DIRECT, INDIRECT,\n * INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT\n * NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE,\n * DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY\n * THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT\n * (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF\n * THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.\n*/",
+      "licenseId" : "LicenseRef-1"
     } ],
-    "annotations" : [ {
-      "annotator" : "Person: Jane Doe ()",
+    "annotation" : [ {
       "annotationDate" : "2010-01-29T18:30:22Z",
       "annotationType" : "OTHER",
+      "annotator" : "Person: Jane Doe ()",
       "comment" : "Document level annotation"
-    } ],
-    "reviewers" : [ {
-      "reviewer" : "Person: Suzanne Reviewer",
-      "reviewDate" : "2011-03-13T00:00:00Z",
-      "comment" : "Another example reviewer."
     }, {
-      "reviewer" : "Person: Joe Reviewer",
-      "reviewDate" : "2010-02-10T00:00:00Z",
+      "annotationDate" : "2010-02-10T00:00:00Z",
+      "annotationType" : "REVIEW",
+      "annotator" : "Person: Joe Reviewer",
       "comment" : "This is just an example.  Some of the non-standard licenses look like they are actually BSD 3 clause licenses"
+    }, {
+      "annotationDate" : "2011-03-13T00:00:00Z",
+      "annotationType" : "REVIEW",
+      "annotator" : "Person: Suzanne Reviewer",
+      "comment" : "Another example reviewer."
+    } ],
+    "documentDescribes" : [ "SPDXRef-File", "SPDXRef-Package" ],
+    "packages" : [ {
```

Comment:
Unless I'm just misreading it, I think the packages are missing their SPDX ID definitions? e.g. I see that "documentDescribes" above references SPDXRef-Package, and so do relationships below, but I'm not seeing which package that ID actually corresponds to.

Reply:
That would be a bug - The SPDX ID's are supposed to be included for all SPDX items.  I'll work on this and update the examples.

Thanks for the catch!

Reply:
Added SPDX ID's in the latest commit